packages/graphql-platform-connector-mariadb/src/graphql-platform/connector/database/table/column.ts

```diff
@@ -236,6 +239,11 @@ export class Column {
       : undefined;
   }
 
+  @Memoize()
+  public get fullText(): boolean {
```

Comment:
Unfortunatly, `getFlagValue` is only for `flag` / `boolean`, you must do it yourself here:

```ts
@Memoize()
public get fullText(): number | null {
  return Number.isInteger(this.config.fullText) && this.config.fullText > 0 ? this.config.fullText : null;
}
```